modules/storage-persistent-storage-aws-maximum-volumes.adoc

```diff
@@ -7,13 +7,17 @@
 
 By default, {product-title} supports a maximum of 39 EBS volumes attached to one
 node. This limit is consistent with the
-link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS
-volume limits].
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS volume limits].
+
+[IMPORTANT]
+====
+As a cluster administrator, you must use either in-tree volumes or Container Storage Interface (CSI) volumes, but never both at the same time. The maximum attached EBS volume number is counted separately for in-tree and CSI volumes.
+====
 
 {product-title} can be configured to have a higher limit by setting the
 environment variable `KUBE_MAX_PD_VOLS`. However, AWS requires a particular
```

Comment:
Is there a doc about how to set the KUBE_MAX_PD_VOLS env variable can be linked here?

Reply:
Discussed this with scheduler QE, looks we don't support KUBE_MAX_PD_VOLS in 4.x. @jsafrane Could you help double confirm?

Reply:
I removed this note about `KUBE_MAX_PD_VOLS` but let me know if that is incorrect @jsafrane.